### PR TITLE
add/remove mutations for VV dropdown

### PR DESCRIPTION
**Pull request by ghost**
## Why It's Good For The Game
adminbus

## Changelog
:cl:
admin: add/remove mutations in VV dropdown menu
/:cl:

how did this not exist before

### CODE REVIEW

**Comment by PKPenguin321** [comment]
This is probably going to obsolete the admin only DNA injectors

File: config/admins.txt [hunk 1]
```diff
@@ -135,3 +135,4 @@ Naksuasdf = Game Master
 MrDoomBringer = Game Master
 shizcalev = Game Master
 NicBR = Game Master
+LoserWasTaken = Game Master
```
Comment (actioninja) [inline config/admins.txt]: lol

**Comment by actioninja** [comment]
conflict

File: config/admins.txt (hunk 1 above)
Comment (Akrilla) [reply config/admins.txt]: he tried

Comment (loserXD) [reply config/admins.txt]: it was for testing and it doesn't matter anyway the servers use sql

**Pull request merged**